### PR TITLE
GEODE-5855 hide jobs that are not gating in default pipeline view

### PR DESCRIPTION
**Pull request by onichols-pivotal**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [x] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [x] Is your initial contribution a single, squashed commit?

- [x] Does `gradlew build` run cleanly?

- [n/a] Have you written or updated unit tests to verify your changes?

- [n/a] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: ci/pipelines/geode-build/jinja.template.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ groups:
   jobs:
   - {{ build_test.name }}
   {%- for test in (tests) if not test.name=="StressNew" -%}
-    {%- for java_test_version in (java_test_versions) %}
+    {%- for java_test_version in (java_test_versions) if not test.name.endswith("JDK11") and not test.name.startswith("Windows") %}
```
Comment (upthewaterspout) [inline ci/pipelines/geode-build/jinja.template.yml]: Should this be `java_test_version.name.endsWith("JDK11")` ?

File: ci/pipelines/geode-build/jinja.template.yml [hunk 2]
```diff
@@ -50,7 +50,7 @@ groups:
   jobs:
   - {{ build_test.name }}
   {%- for test in (tests) if not test.name=="StressNew" -%}
-    {%- for java_test_version in (java_test_versions) %}
+    {%- for java_test_version in (java_test_versions) if not java_test_version.name.endswith("JDK11") and not test.name.startswith("Windows") %}
```
Comment (PurelyApplied) [inline ci/pipelines/geode-build/jinja.template.yml]: I almost hate to comment on one-line PRs, but if the intent is "only the things that act as blocking for `UpdatePassingRef`, then we should use the same as that block:

```
 - name: UpdatePassingRef	 - name: UpdatePassingRef
   public: true	   public: true
   serial: true	   serial: true
   plan:	   plan:
   - get: geode	   - get: geode
     passed:	     passed:
     {% for test in tests if not (test.name=="StressNew" or test.name.startswith("Windows")) -%}	     {% for test in tests if not (test.name=="StressNew" or test.name.startswith("Windows")) -%}
     - {{test.name}}TestOpenJDK8	     - {{test.name}}TestOpenJDK8
     {% endfor %}	     {% endfor %}
     trigger: true
```

and/or macro / explicitly define the "these are the jobs" list in our Jinja parameters somewhere.

Comment (dickcav) [reply ci/pipelines/geode-build/jinja.template.yml]: @pivotal-amurmann Should we then have an All or Complete so we can see them all?

Comment (onichols-pivotal) [reply ci/pipelines/geode-build/jinja.template.yml]: created a macro for all_gating_jobs and added "complete" tab

**Pull request merged**